setup.py

```diff
@@ -4,15 +4,13 @@
     long_description = fh.read()
 
 dependencies = [
-    # "chia-blockchain @ git+https://github.com/Chia-Network/chia-blockchain.git@1.6.2",
-    "chia-blockchain==1.7.0",
-    "packaging==21.3",
+    "chia-blockchain>=1.8.0",
```

Comment:
`chia-blockchain` doesn't make backwards compatible guarantees so `>=` is not generally a good idea.  If you have concluded this is compatible with 1.8.0 and 1.8.1 then you can do this.

```suggestion
    "chia-blockchain>=1.8.0,<=1.8.1",
```

I don't know that it is generally worthwhile though with `chia-blockchain`.

```suggestion
    "chia-blockchain==1.8.1",
```